src/app/checkout/Checkout.tsx

```diff
@@ -66,6 +66,7 @@ export interface CheckoutProps {
 
 export interface CheckoutState {
     activeStepType?: CheckoutStepType;
+    billingSameAsShipping: boolean;
```

Comment:
Should this have the `is` prefix as per other booleans in this interface?

Reply:
Good idea. I've added `is` prefix to it